applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -292,6 +292,10 @@ module = AUDIO_SYNC_TIMER
 module-str = audio-sync-timer
 source "subsys/logging/Kconfig.template.log_config"
 
+module = LE_AUDIO_RX
```

Comment:
Should be changed if the file names are changed

Reply:
Discuss